### PR TITLE
feat(jobs): make the job list pretty

### PR DESCRIPTION
**Pull request by hutchic**
plan is to use a bootstrap modal to display the posting details when the visitor clicks on the job listing

job_listings

### CODE REVIEW

File: jobs/index.html [hunk 1]
```diff
@@ -18,8 +22,12 @@
...
 
   var jobContainer = document.getElementById('jobs')
-  var jobItem = function (contents) {
-    return '<li>' + contents + '</li>'
+  var jobItem = function (element) {
+    date = new Date(element.created_at)
+    var dateOptions = {
+      year: "numeric", month: "short", day: "numeric"
+    };
+    return '<a class="list-group-item"><h4><span class="job-title">' + element.title + '</span><span class="badge pull-right">' + date.toLocaleDateString("en-US", dateOptions) + '</span></h4></a>'
```
Comment (hutchic) [inline jobs/index.html:30]: this needs to be cleaned up to use string interpolation better or just something ... input appreciated (cc @theetrain )

**Comment by theetrain** [comment]
I think rather than bootstrap, let's use Material Lite since it's already used on the site. https://getmdl.io/components/index.html#dialog-section

File: jobs/index.html (hunk 1 above)
Comment (theetrain) [reply jobs/index.html:30]: I don't think we're allowed to wrap an `h4` within an `a`. We'll clean this up.

Comment (hutchic) [reply jobs/index.html:30]: I was shamelessly stealing the design from here http://demo.jobskee.com/

**Pull request merged**